### PR TITLE
Update to the vertx opentracing example to use the standard 'database…

### PR DESCRIPTION
**Pull request by objectiser**
….statement' property, and use the constrant in tests

### CODE REVIEW

**Comment by objectiser** [comment]
@pavolloffay Could you review this please?

Reviewer notes:
* Just bring the example (and some tests) in line to use the standard 'database.statement' property name, and update the tests to use the constant.

File: examples/vertx-opentracing/src/main/java/org/hawkular/apm/examples/vertx/opentracing/AccountManager.java [hunk 1]
```diff
@@ -69,7 +69,7 @@ protected void init(EventBus eb, Tracer tracer) {
                     try (Span retrieveAccountSpan = tracer.buildSpan("RetrieveAccount")
                             .asChildOf(getAccountSpan)
                             .withTag("database.url", "AccountsDB")
-                            .withTag("sql", "SELECT .....")
+                            .withTag("database.statement", "SELECT account FROM Accounts WHERE id = ?")
```
Comment (pavolloffay) [inline examples/vertx-opentracing/src/main/java/org/hawkular/apm/examples/vertx/opentracing/AccountManager.java]: use `Constants.PROP_DATABASE_STATEMENT`?

File: examples/vertx-opentracing/src/main/java/org/hawkular/apm/examples/vertx/opentracing/InventoryManager.java [hunk 2]
```diff
@@ -74,7 +74,7 @@ protected void initGetItemConsumer(EventBus eb, Tracer tracer) {
                     try (Span queryInventorySpan = tracer.buildSpan("QueryInventory")
                             .asChildOf(getItemSpan)
                             .withTag("database.url", "InventoryDB")
-                            .withTag("sql", "SELECT item FROM InventoryTable WHERE id = :id")
+                            .withTag("database.statement", "SELECT item FROM Inventory WHERE id = ?")
```
Comment (pavolloffay) [inline examples/vertx-opentracing/src/main/java/org/hawkular/apm/examples/vertx/opentracing/InventoryManager.java]: use `Constants.PROP_DATABASE_STATEMENT`?

File: examples/vertx-opentracing/src/main/java/org/hawkular/apm/examples/vertx/opentracing/InventoryManager.java [hunk 3]
```diff
@@ -105,7 +105,7 @@ protected void initOrdersConfirmedConsumer(EventBus eb, Tracer tracer) {
                 try (Span queryInventorySpan = tracer.buildSpan("WriteInventory")
                         .asChildOf(orderConfirmedSpan)
                         .withTag("database.url", "InventoryDB")
-                        .withTag("sql", "UPDATE FROM InventoryTable ....")
+                        .withTag("database.statement", "UPDATE Inventory SET item=?")
```
Comment (pavolloffay) [inline examples/vertx-opentracing/src/main/java/org/hawkular/apm/examples/vertx/opentracing/InventoryManager.java]: use `Constants.PROP_DATABASE_STATEMENT`?

File: examples/vertx-opentracing/src/main/java/org/hawkular/apm/examples/vertx/opentracing/OrderLog.java [hunk 4]
```diff
@@ -63,7 +63,7 @@ protected void initOrdersConfirmedConsumer(EventBus eb, Tracer tracer) {
                 try (Span storeOrderSpan = tracer.buildSpan("WriteOrder")
                         .asChildOf(orderConfirmedSpan)
                         .withTag("database.url", "OrdersDB")
-                        .withTag("sql", "UPDATE .....")
+                        .withTag("database.statement", "UPDATE Orders SET order=?")
```
Comment (pavolloffay) [inline examples/vertx-opentracing/src/main/java/org/hawkular/apm/examples/vertx/opentracing/OrderLog.java]: use `Constants.PROP_DATABASE_STATEMENT`?

File: examples/vertx-opentracing/src/main/java/org/hawkular/apm/examples/vertx/opentracing/OrderLog.java [hunk 5]
```diff
@@ -94,7 +94,7 @@ protected void initGetOrdersConsumer(EventBus eb, Tracer tracer) {
                 try (Span retrieveOrdersSpan = tracer.buildSpan("RetrieveOrders")
                         .asChildOf(getOrdersSpan)
                         .withTag("database.url", "OrdersDB")
-                        .withTag("sql", "SELECT .....")
+                        .withTag("database.statement", "SELECT order FROM Orders WHERE accountId = ?")
```
Comment (pavolloffay) [inline examples/vertx-opentracing/src/main/java/org/hawkular/apm/examples/vertx/opentracing/OrderLog.java]: use `Constants.PROP_DATABASE_STATEMENT`?

**Comment by objectiser** [comment]
@pavolloffay your comments relate to the example opentracing app, so didn't want it to reference any Hawkular APM classes.

**Comment by pavolloffay** [comment]
Is there a similar key for a database statement in opentracing jar?

**Comment by objectiser** [comment]
no they haven't defined any conventions yet - but hopefully will do at some point, if it makes sense to standardise them across opentracing implementations.

**Comment by pavolloffay** [comment]
It can be merged, But we have to be careful with refactoring if we rename the key.

**Comment by objectiser** [comment]
@pavolloffay thanks! yes agree.

**Pull request merged**